src/topology/category/Top/limits.lean

```diff
@@ -42,6 +42,21 @@ def limit_cone (F : J ⥤ Top.{u}) : cone F :=
       continuous_to_fun := show continuous ((λ u : Π j : J, F.obj j, u j) ∘ subtype.val),
         by continuity } } }
 
+/--
+A choice of limit cone for a functor `F : J ⥤ Top` whose topology is defined as an
+infimum of topologies infimum.
+Generally you should just use `limit.cone F`, unless you need the actual definition
+(which is in terms of `types.limit_cone`).
+-/
+def limit_cone_infi (F : J ⥤ Top.{u}) : cone F :=
```

Comment:
```suggestion
@[simps] def limit_cone_infi (F : J ⥤ Top.{u}) : cone F :=
```

Reply:
I would prefer to stay consistent with `limit_cone` here, which doesn't have a `@[simps]`. I think this is for a good reason, since usually one should indeed just use `limit.cone F`.

Reply:
Agreed.